src/js/view/controls/controls.ts

```diff
@@ -7,7 +7,7 @@ import { now } from 'utils/date';
 import button from 'view/controls/components/button';
 import Controlbar from 'view/controls/controlbar';
 import DisplayContainer from 'view/controls/display-container';
-import NextUpToolTip from 'view/controls/nextuptooltip';
+import NextUpTooltip from 'view/controls/nextuptooltip';
```

Comment:
I just renamed this and the `nextUpTooltip` because the uppercase T was bothering me. Can revert.